CodeListLibrary_project/clinicalcode/entity_utils/search_utils.py

```diff
@@ -67,7 +67,10 @@ def get_renderable_entities(request, entity_type=None):
         }
 
     template_ids = list(templates.values_list('id', flat=True))
-    entities = PublishedGenericEntity.objects.filter(template__in=template_ids)
+
+    entities = GenericEntity.objects.filter(template__in=template_ids).all()
+    entities = PublishedGenericEntity.objects.filter(entity__in=entities).order_by('-created').distinct().order_by('id')
```

Comment:
We need to look into removing PublishedGenericEntity, and all 'published' models in general.

The same could be achieved using the `status` field and `history` objects and it increases the complexity of getting a meaningful read from the DB